src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerBackpack.java

```diff
@@ -160,7 +161,7 @@ public void setSize(int size) {
      */
     public void save() {
         for (int i = 0; i < size; i++) {
-            cfg.setValue("backpacks." + id + ".contents." + i, inventory.getItem(i));
+            cfg.setValue(PlayerBackpack.backpacks + id + ".contents." + i, inventory.getItem(i));
```

Comment:
`PlayerBackpack` is redundant here, no need to specify the class you are already in.

Reply:
ff83a0183